node/sc/subbridge.go

```diff
@@ -206,6 +209,37 @@ func NewSubBridge(ctx *node.ServiceContext, config *SCConfig) (*SubBridge, error
 	return sc, nil
 }
 
+func (sb *SubBridge) SetRPCConn(conn net.Conn) {
+	sb.rpcConn = conn
+	go func() {
+		for {
+			data := make([]byte, rpcBufferSize)
```

Comment:
How about add quit routine by `<-sc.quitSync:`?

Reply:
Update error handling for the rpc client. PTAL.